components/omega/src/ocn/auxiliaryVars/TracerAuxVars.h

```diff
@@ -0,0 +1,101 @@
+#ifndef OMEGA_AUX_TRACER_H
+#define OMEGA_AUX_TRACER_H
+
+#include "DataTypes.h"
+#include "HorzMesh.h"
+#include "OmegaKokkos.h"
+#include "auxiliaryVars/LayerThicknessAuxVars.h"
+
+#include <string>
+
+namespace OMEGA {
+
+class TracerAuxVars {
+ public:
+   Array3DReal HTracersOnEdge;
+   Array3DReal Del2TracersOnCell;
+
+   FluxThickEdgeOption TracersOnEdgeChoice = Center;
```

Comment:
Is this hard-coded right now, to be added as a config option later?

Reply:
Correct, this option will be set by the config in the follow up PR that integrates these terms and auxvars into the timestep